common/constants/big.go

```diff
@@ -12,12 +12,25 @@ var (
 	Big0 = big.NewInt(0)
 	// Big1 is 1 represented as type "math/big"
 	Big1 = big.NewInt(1)
+	// Big2Exp232 is 2^232 respresented as type "math/big"
```

Comment:
I see how these are slightly convenient, but I probably will avoid using them because they obfuscate the code. 

`b := big.NewInt(1) + util.BigExp(2, 232)` 👈 pretty clear what I'm doing
`b := Big1 + Big2Exp232` 🤔